lib/ClusterShell/CLI/Clush.py

```diff
@@ -977,6 +996,18 @@ def main():
     task.set_info("debug", config.verbosity >= VERB_DEBUG)
     task.set_info("fanout", config.fanout)
 
+    if options.sudo:
+        if config.sudo_command is None:
+            sudo_cmd = 'sudo -S -p "\'\'"'
```

Comment:
i don't think that's the way we deal with default values

Reply:
indeed! thx and sorry, I am a bit rusty!